ProjectMars/Pages/ProfilePage.cs

```diff
@@ -0,0 +1,275 @@
+        private IWebElement skillTab => webDriver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[1]/a[2]"));
+        private IWebElement addNewButton => webDriver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/thead/tr/th[3]/div"));
+        private IWebElement addLanguageTextBox => webDriver.FindElement(By.XPath("//input[@placeholder='Add Language']"));
+        private IWebElement languageLevelDropdown => webDriver.FindElement(By.XPath("//select[@name='level']"));
+        private IWebElement chooseBasicLanguageLevel => webDriver.FindElement(By.CssSelector("option[value='Basic']"));
+        private IWebElement chooseFluentLanguageLevel => webDriver.FindElement(By.CssSelector("option[value='Fluent']"));
+        private IWebElement chooseConversationalLanguageLevel => webDriver.FindElement(By.CssSelector("option[value='Conversational']"));
+        private IWebElement chooseNBlLanguageLevel => webDriver.FindElement(By.CssSelector("option[value='Native/Bilingual']"));
+        private IWebElement addButtonLanguage => webDriver.FindElement(By.XPath("//input[@class='ui teal button']"));
+        private IWebElement newLanguage => webDriver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/tbody[last()]/tr/td[1]"));
+        private IWebElement newLevel => webDriver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/tbody[last()]/tr/td[2]"));
+        private IWebElement editButton => webDriver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/tbody/tr/td[3]/span[1]/i"));
+        private IWebElement editTextBox => webDriver.FindElement(By.XPath("//input[@placeholder='Add Language']"));
+        private IWebElement updateButton => webDriver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/tbody[last()]/tr/td/div/span/input[1]"));
+        private IWebElement newEditLanguage => webDriver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/tbody[last()]/tr/td[1]"));
+        private IWebElement newEditLevel => webDriver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/tbody[last()]/tr/td[2]"));
+        private IWebElement deleteButton => webDriver.FindElement(By.XPath("//*[@id=\"account-profile-section\"]/div/section[2]/div/div/div/div[3]/form/div[2]/div/div[2]/div/table/tbody[last()]/tr/td[3]/span[2]/i"));
+
+
+        public void GotoLanguageTab(IWebDriver driver)
```

Comment:
you are passing the driver to the method this is not needed as you have already passed the driver in the constructor please fix this